src/common/routes/catalog.ts

```diff
@@ -30,4 +30,10 @@ export const catalogRoute: RouteProps = {
   path: "/catalog/:group?/:kind?"
 };
 
+export const getPreviousTabUrl = (path: string) => {
+  return `${catalogURL()}/${path || browseCatalogTab}`;
+};
```

Comment:
@ixrock Left url constructor untouched for the sake of backward compatibility. Users already has this form saved in their local storage. And `to` prop inside `<Redirect/>` can take `string` safely.

Reply:
Let's say we will change this route path in future to just:

```ts
export const catalogRoute: RouteProps = {
  path: "/catalog"
};
```
And will serve params like `/catalog?group=someGroup&kind=kind`, wouldn't it break backward compatibility with that users?

In other words I mean it's better and safer to parse already that params and serve separately in used URLs.
But `Redirect` can and should accept strings of course :)